plugins/modules/nodebalancer.py

```diff
@@ -342,9 +346,10 @@ def _create_nodebalancer(self) -> Optional[NodeBalancer]:
         params = self.module.params
         label = params.get("label")
         region = params.get("region")
+        firewall_id = params.get("firewall_id")
 
         try:
-            return self.client.nodebalancer_create(region, label=label)
+            return self.client.nodebalancer_create(region, label=label, firewall_id=firewall_id)
```

Comment:
We'll probably want to implement update or validation logic to prevent users from updating the `firewall_id` and expecting that a change has been applied.

This flow could look something like:
1. If `firewall_id` isn't specified, skip
2. Get the firewall that `firewall_id` is pointing to, raise an error if it doesn't exist
3. Check that there is a device under the firewall pointing to the NodeBalancer, if not raise a "non-updatable" error